HACKING.md

```diff
@@ -0,0 +1,118 @@
+## Introduction
+LU is build with the [Entity–component–system](https://en.wikipedia.org/wiki/Entity_component_system) which you may know from the Unity Engine. In short that means everything that you see on screen (and more) is a GameObject on which Components can be attached. These components give the object functionality like rendering, physics and so on. 
+
+LU used two different types of servers to communicate. These are: 
+* The authentication server (Uchu.Auth), which handles login and redirection to the world server. 
+* The world server, which handles two things
+  * Character selection (Uchu.Char)
+  * Main game (Uchu.World)
+
+**Note:** A world server in Uchu only handles one world but there can be multiple world servers in one Uchu instance. They can even serve the same map if one world server has reached the maximum amount of player. 
+
+The address of the auth server is configured in the `boot.cfg` file in the installation directory of the client. All other servers can have any address because the previous server sends the address to the client.
+
+There were also two other server types that aren't implemented in Uchu. These are
+* The Chat server which was never used because the world sever handles chat messages
```

Comment:
Add a comma after 'server'

Reply:
I'd maybe also reword this to say "The Chat server, which isn't used, since chat messages are handled in the world server."

Reply:
I never really understood where to place commas in English